.travis/linux_travis_arm64_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Basic tools
+
+set -x
+
+if [ ! -z "${DOCKERHUB_PULL_USERNAME:-}" ]; then
+  set +x
+  echo "${DOCKERHUB_PULL_PASSWORD}" | docker login --username "${DOCKERHUB_PULL_USERNAME}" --password-stdin
+  set -x
+fi
+
+sudo apt-get update -qq
+sudo apt-get install -qq mysql-client zip jq expect nfs-kernel-server build-essential curl git libnss3-tools libcurl4-gnutls-dev docker-compose
+
+# Copy docker-compose to /usr/local/bin because Travis' pre-installed version leads to exec format error
+sudo cp /usr/bin/docker-compose /usr/local/bin/docker-compose
```

Comment:
Is this Ubuntu 20.04? I found on Raspberry Pi (Debian 10) that the only way to get a working docker-compose was to install with `pip3 install docker-compose`; the bundled one has problems, and there is no other distribution path. But I guess if this is Ubuntu 20.04... the apt version works there.

Reply:
Ah, I do see it's focal, thanks.